src/python/dart/engine/emr/metadata.py

```diff
@@ -14,6 +14,32 @@ class EmrActionTypes(object):
                     'description': 'The contents of this script will be executed as a bootstrap step',
                     "x-schema-form": {"type": "textarea"}
                 },
+                'configurations': {
```

Comment:
Two things left to do here:

1. See if our tooling can correctly resolve the ref to EMRConfiguration
2. Figure out how to expose this in the UI via x-schema-form